x/auction/types/auctions.go

```diff
@@ -66,15 +68,35 @@ func (a BaseAuction) GetType() string { return "base" }
 
 // Validate verifies that the auction end time is before max end time
 func (a BaseAuction) Validate() error {
-	if a.EndTime.After(a.MaxEndTime) {
-		return fmt.Errorf("MaxEndTime < EndTime (%s < %s)", a.MaxEndTime, a.EndTime)
+	// ID can be 0 for surplus, Debt and collateral auctions
+	if strings.TrimSpace(a.Initiator) == "" {
+		return errors.New("auction initiator cannot be blank")
 	}
 	if !a.Lot.IsValid() {
 		return fmt.Errorf("invalid lot: %s", a.Lot)
 	}
+	// NOTE: bidder can be nil for Surplus and Collateral auctions
+	if a.Bidder != nil && a.Bidder.Empty() {
+		return errors.New("auction bidder cannot be empty")
+	}
+	if a.Bidder != nil && len(a.Bidder) != sdk.AddrLen {
+		return fmt.Errorf("the expected bidder address length is %d, actual length is %d", sdk.AddrLen, len(a.Bidder))
+	}
 	if !a.Bid.IsValid() {
 		return fmt.Errorf("invalid bid: %s", a.Bid)
 	}
+	if a.EndTime.IsZero() || a.MaxEndTime.IsZero() {
+		return errors.New("end time cannot be zero")
+	}
+	if a.EndTime.After(a.MaxEndTime) {
+		return fmt.Errorf("MaxEndTime < EndTime (%s < %s)", a.MaxEndTime, a.EndTime)
+	}
+	if a.HasReceivedBids && !a.Bid.IsPositive() {
+		return fmt.Errorf("cannot have a zero value bid when auction has a true HasReceivedBids flag value")
+	}
+	if !a.HasReceivedBids && a.Bid.IsPositive() {
```

Comment:
This check is failing on sims. `auction/simulation/genesis.go` needs to be updated to set `HasReceivedBids` to be true for the auction that is created at genesis.